main.go

```diff
@@ -106,11 +107,12 @@ func main() {
 	}
 
 	upgradeLogger := ctrl.Log.WithName("controllers").WithName("Upgrade")
-
+	clusterUpdateStateManager := upgrade.NewClusterUpdateStateManager(upgradeLogger, mgr.GetClient())
```

Comment:
you could pass here : upgradeLogger.WithName("manager")
so its clear from logs which component did the logging. WDYT?